tests/test_node.py

```diff
@@ -14,206 +14,208 @@ class Box(BaseBox):
         pass
 
     def layout(self, root, viewport):
-        # A simple layout scheme that allocats twice the viewport size.
+        # A simple layout scheme that allocates twice the viewport size.
         root.layout.content_width = viewport.width * 2
         root.layout.content_height = viewport.height * 2
 
 
-class NodeTests(TestCase):
-    def setUp(self):
-        self.maxDiff = None
+def test_create_leaf():
+    """A leaf can be created"""
+    style = Style()
+    leaf = Node(style=style)
 
-    def test_create_leaf(self):
-        "A leaf can be created"
-        style = Style()
-        leaf = Node(style=style)
+    assert leaf._children is None
+    assert leaf.children == []
+    assert not leaf.can_have_children
 
-        self.assertIsNone(leaf._children)
-        self.assertEqual(leaf.children, [])
-        self.assertFalse(leaf.can_have_children)
+    # An unattached leaf is a root
+    assert leaf.parent is None
+    assert leaf.root == leaf
 
-        # An unattached leaf is a root
-        self.assertIsNone(leaf.parent)
-        self.assertEqual(leaf.root, leaf)
+    # A leaf can't have children
+    child = Node(style=style)
 
-        # A leaf can't have children
-        child = Node(style=style)
+    with pytest.raises(ValueError):
+        leaf.add(child)
 
-        with self.assertRaises(ValueError):
-            leaf.add(child)
 
-    def test_create_node(self):
-        "A node can be created with children"
-        style = Style()
+def test_create_node():
+    """A node can be created with children"""
+    style = Style()
 
-        child1 = Node(style=style)
-        child2 = Node(style=style)
-        child3 = Node(style=style)
+    child1 = Node(style=style)
+    child2 = Node(style=style)
+    child3 = Node(style=style)
 
-        node = Node(style=style, children=[child1, child2, child3])
+    node = Node(style=style, children=[child1, child2, child3])
 
-        self.assertEqual(node.children, [child1, child2, child3])
-        self.assertTrue(node.can_have_children)
+    assert node.children, [child1, child2 == child3]
```

Comment:
Pretty sure this isn't doing what you think. It's passing because `node.children` is not empty; but it's not comparing that the list of children is three elements long... or that the list of children is `[child1, False]`.

Reply:
Right you are! That's a search-and-replace flub from `self.assertEqual(node.children, [child1, child2, child3])`. Should be `assert node.children == [child1, child2, child3]`.